jrnl/Entry.py

```diff
@@ -49,8 +49,8 @@ def tags(self):
 
     @staticmethod
     def tag_regex(tagsymbols):
-        pattern = fr'(?u)(?:^|\s)([{tagsymbols}][-+*#/\w]+)'
-        return re.compile(pattern)
+        pattern = r'(?<!\S)([{tags}][-+*#/\w]+)'.format(tags=tagsymbols)
+        return re.compile(pattern, re.UNICODE)
```

Comment:
Same with some of these changes. re.UNICODE is not needed and you reverted the fstring.

Reply:
I was under the impression that `(?u)` and `re.UNICODE` were the same, so that was intentional. Otherwise, these were merging errors. Sorry about that.

Reply:
That's true actually, they are the same and you can safely remove both.

Reply:
Initially, it was done in this commit: 95b23d433c124de685a1bd9891fdff8f1edb308a

Fixed all the merge errors. Sorry about that.